content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -737,6 +738,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
   for fsGroups when mounting a volume in a Pod. See
   [Configure volume permission and ownership change policy for Pods](/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods)
   for more details.
+- `ContextualLogging`: Enables more informative log output in components that
+  use the [corresponding logging
+  calls](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging#feature-gate).
```

Comment:
Is there any documentation on https://k8s.dev/ about contextual logging? Ideally, we'd make sure that there was.

Reply:
Not yet, because it currently doesn't make a difference for end-users.

Reply:
See content/en/docs/concepts/cluster-administration/system-logs.md for an explanation based on the example command.